src/controllers/user.controller.js

```diff
@@ -0,0 +1,15 @@
+import { getRoverPhoto } from '../services/rover.service.js';
+import { processUser } from '../services/user.service.js';
+
+export const postUser = async (req, res, next) => {
+  try {
+    processUser(req);
+
+    const roverPhotoUrl = await getRoverPhoto(req.body.api_key);
```

Comment:
I think we should not pass api_key as request, we should take it from env variable while making a get request to NASA

Reply:
In this case, the request body containing the id, name and API key will be useless 😢 
I thought it would be nice to use some of the data from it

Reply:
Only if you want to do like this in the educational purposes :)